ui/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { CatIcon, CertificateIcon, BuildIcon, DomainIcon, UserIcon } from '@patternfly/react-icons';
+import './Icon.css';
+
+interface Name {
+  name: string;
+}
+
+const Icon: React.FC<Name> = ({ name }) => {
+  switch (name) {
+    // Support Tiers
+    case 'Official':
+      return <CatIcon size="sm" color="#484848" className="Icon" label={name} />;
+    case 'Verified':
+      return <CertificateIcon size="sm" color="#484848" className="Icon" label={name} />;
+    case 'Community':
+      return <UserIcon size="sm" color="#484848" className="Icon" label={name} />;
+    // Kinds
+    case 'Task':
+      return <BuildIcon size="sm" color="black" className="Icon" label={name} />;
```

Comment:
I think the color of kind is also `#484848` on Hub and hence we can also move the color to the css class defined

Reply:
on Hub the color of Kind is black that's why I added that here https://github.com/tektoncd/experimental/blob/51b94680d7fd0eb17e355a8709545c083c4270a3/hub/ui/src/components/filter/Filter.tsx#L95
but will change this if you want